### DISCUSSION TITLE
.doc and .xls attachments?

### DISCUSSION THREAD

**Discussion by E-JOKE** [Budibase/budibase#10027] [Help] [answered]
I'm using a Budibase Database in which users can upload .pdf, .doc of .xls attachments, that are mostly stored on their Google Drives.  The uploaded .pdf files can be opened in Budibase without any problem, but the uploaded .doc and .xls files just will not open (a screen is shown that tries to save these files on the local drive under another name). 
I can't find anything about this in the helpfiles or forums. Am I missing something?

**Comment by ConorWebb96** [comment]
Hey @E-JOKE,

I don't think I understand fully, is the issue with redownloading the files? I've just done a test and was able to upload and download the files stored through Budibase. (See video below)

https://user-images.githubusercontent.com/126772285/225250973-b71b6ce1-08ad-4bc4-ba46-4d6d1d2f2c69.mp4

Hopefully this helps!

**Comment by E-JOKE** [comment]
Hi Conor,

 

The problem is not that I cannot redownload the files.
The problem is that I want the files to show on screen when I click them. This works with .pdf files, but not with .doc or .xls (attachment included).

 

Even better would be if the attachments remain in the Google Drive of the users and that only a link to a shared Google Drive is saved in the Budibase database. 
I tried this method, but cannot get this to work.

 

 

Regards,

Erik 

 

 

Van: Conor Webb ***@***.***> 
Verzonden: woensdag 15 maart 2023 10:07
Aan: Budibase/budibase ***@***.***>
CC: E-JOKE ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [Budibase/budibase] .doc and .xls attachments? (Discussion #10027)

 

Hey @E-JOKE <https://github.com/E-JOKE> ,

I don't think I understand fully, is the issue with redownloading the files? I've just done a test and was able to upload and download the files stored through Budibase. (See video below)

https://user-images.githubusercontent.com/126772285/225250973-b71b6ce1-08ad-4bc4-ba46-4d6d1d2f2c69.mp4

Hopefully this helps!

**Comment by ConorWebb96** [comment, marked as answer]
Hey @E-JOKE,

For doc/xls files, it isn't possible to see them on the browser through Budibase. These will automatically be downloaded instead.

I will say you could very easily include a google drive doc link so that when clicked it will redirect to your doc hosted there. (I've included some screenshots below to show this.)

<img width="1198" alt="Screenshot 2023-03-16 at 09 58 45" src="https://user-images.githubusercontent.com/126772285/225584789-00462979-9d8b-4168-a4e5-8bc017e98e50.png">

<img width="1342" alt="Screenshot 2023-03-16 at 10 09 32" src="https://user-images.githubusercontent.com/126772285/225584815-a71bfcc3-27fd-4c22-8f6e-2f4fc4b32e96.png">


I hope this helps!
